src/olympia/editors/helpers.py

```diff
@@ -79,67 +79,6 @@ def editor_page_title(context, title=None, addon=None):
 
 @register.function
 @jinja2.contextfunction
-def editors_breadcrumbs(context, queue=None, addon=None, items=None,
```

Comment:
Didn't you log in to the admin tools/editor tools locally and test these were hidden? They seem useful if not hidden (and restyle might not have touched the editor tools as much).

If they were hidden or @jvillalobos says these aren't needed it's fine.